visualizations/frontend/other/TTable.vue

```diff
@@ -66,199 +66,217 @@
-            >
-              <div class="px-4">
+              <!-- Radio buttons -->
+              <input
+                v-if="showRadioButtons"
+                :id="rindex"
+                :ref="'radio_' + rindex"
+                v-model="internalSelectedItem"
+                type="radio"
+                :value="row.originalRow"
+              />
+
+              <!-- Check boxes -->
+              <input
+                v-if="showCheckboxes"
+                :id="rindex"
+                :ref="'check_' + rindex"
+                v-model="internalSelectedItems"
+                type="checkbox"
+                :value="row.originalRow"
+              />
+
+              <!-- Columns for the row -->
+              <div
+                v-for="(column, cindex) in internalColumns"
+                :key="column.property"
+                :ref="'rowCell_' + gindex + '_' + rindex + '_' + cindex"
+                class="align-top cellPadding"
+                :class="generateCellClasses({ column, cindex, row, rindex })"
+                @click="
+                  ($event) =>
+                    handleCellClick(getCellValue(row, column), row, column)
+                "
+              >
                 <slot
-                  class="detail-row-slot"
-                  name="detail_row_slot"
-                  v-bind="row.originalRow"
-                ></slot>
+                  :name="column.property"
+                  :row="row.originalRow"
+                  :value="getRawCellValue(row, column)"
+                  :rendered_value="getCellValue(row, column)"
+                  :rendered="getRenderedCellValue(row, column)"
+                >
+                  {{ getCellValue(row, column) }}
+                </slot>
+              </div>
+
+              <!-- Detail row -->
+              <div
+                v-if="!canCollapseDetailRows || row.detailRowOpen"
+                :key="'detailRow' + rindex"
+                class="spanAllColumns"
+              >
+                <div class="px-4">
+                  <slot
+                    class="detail-row-slot"
+                    name="detail_row_slot"
+                    v-bind="row.originalRow"
+                  ></slot>
+                </div>
               </div>
             </div>
           </div>
         </div>
       </div>
-      <div v-if="!isPdf && (canPage || canPageServer)" class="spanAllColumns">
-        <SnykPager
-          id="pagingControls"
-          :start-index="startIndex"
-          :end-index="endIndex"
-          :number-of-items="totalRows"
-          :items-per-page="internalRowsPerPage"
-          :items-per-page-options="rowsPerPageOptions"
-          @updateItemsPerPage="updateItemsPerPage"
-          @updateStartIndex="updateStartIndex"
-          @updateEndIndex="updateEndIndex"
-          @setResetFunction="setPagerResetFunction"
-        />
+      <!-- Pagination  -->
+      <div v-if="!isPdf && (canPage || canPageServer)">
```

Comment:
this can be combined back into one div. I thought there was a bug, but it was rather that the `canPageServer` was being passed to the table, even though the element was not showing. It resulted in a double border at the bottom of tables on the ETC Analytics report. The condition may need to be update though, to be not looking at `canPageServer` but rather the condition where we decide to show the Pager (inside the SnykPager) component.

I mentioned that we need to elevate that condition, and I may still do it here.

Reply:
I have now addressed this.